### PR TITLE
[MIG][10.0][html_text] Migrate 

### PR DESCRIPTION
**Pull request by yajo**
@Tecnativa

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Check Travis status

**Review by chienandalu — APPROVED** [review]
Code review ok. Travis fails are just logged exceptions as expected from the tests behavior.

**Comment by pedrobaeza** [comment]
Travis should be green

**Comment by pedrobaeza** [comment]
The same as the other PR

File: html_text/tests/test_extractor.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        with self.assertRaises(etree.XMLSyntaxError):
+            self.text_from_html("", fail=True)
+
+    def test_false_html(self):
+        """``False`` HTML handled correctly."""
+        self.assertEqual(self.text_from_html(False), "")
+        with self.assertRaises(TypeError):
+            self.text_from_html(False, fail=True)
+
+    def test_bad_html(self):
```
Comment (moylop260) [inline html_text/tests/test_extractor.py]: You should use `@tools.mute_logger()` to silent traceback logging expected errors.

Comment (moylop260) [reply html_text/tests/test_extractor.py]:
```diff
+    @odoo.tools.mute_logger('odoo.addons.html_text.models.ir_fields_converter')
    def test_bad_html(self):
```

Or 
```diff
-     with self.assertRaises(etree.ParserError):
+     with self.assertRaises(etree.ParserError), odoo.tools.mute_logger('odoo.addons.html_text.models.ir_fields_converter'):
```

**Review by pedrobaeza — APPROVED** [review]

**Comment by pedrobaeza** [comment]
Please squash your commits to merge.

**Comment by yajo** [comment]
done

**Pull request merged**